src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -344,7 +350,6 @@ class PrisonerIepLevelReviewService(
     )
   }
 
-  @Transactional
```

Comment:
the only existing call to this method is from the `processOffenderEvent` in this class so the transactional annotation has no effect